packages/catalog-search/src/FacetListBase.jsx

```diff
@@ -31,19 +33,22 @@ const FacetListBase = ({
   const { refinementsFromQueryParams, dispatch } = useContext(SearchContext);
 
   const handleInputOnChange = (item) => {
-    if (item.value && facetValueType === 'array') {
+    // if it is desired to load the same attribute data in multiple dropdowns then
+    // customAttribute can be passed to differentiate them.
+    const index = customAttribute || attribute;
```

Comment:
The default value of `customAttribute` is null as per L115. So there will be no change in existing behaviour.